FrontEnd/src/components/CookiesPolicyPage/LinkContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import { NavLink } from 'react-router-dom';
+import styles from './cookiesPolicyComponent.module.css';
+
+const LinkContainer = () => (
+  <div className={styles['link']}>
+          <NavLink
+            to="/privacy-policy/"
+            className={({ isActive }) =>
+              isActive ? styles['link_text__active'] : styles['link_text']
+            }
+          >
+            Privacy Policy<br />
+          </NavLink>
+          <NavLink
+            to="/terms-and-conditions/"
+            className={({ isActive }) =>
+              isActive ? styles['link_text__active'] : styles['link_text']
+            }
```

Comment:
We use it multiple times. Could we move it it the function?

Reply:
done created the function getClassName for this